apps/api/src/app/events/events.controller.ts

```diff
@@ -118,7 +111,7 @@ export class EventsController {
     @Body() body: TriggerEventToAllRequestDto
   ): Promise<TriggerEventResponseDto> {
     const transactionId = body.transactionId || uuidv4();
-    const mappedActor = body.actor ? this.mapActor(body.actor) : null;
+    const mappedActor = typeof body.actor === 'string' ? { subscriberId: body.actor } : null;
```

Comment:
make the mapping in order to remove the dep of mapTriggerRecipients from the API.

Reply:
```suggestion
    const mappedActor = typeof body.actor === 'string' ? { subscriberId: body.actor } : body.actor ;
```

Shouldn't it be like this? in case the actor was passed as an object of `{ subscriberId: body.actor }`

Reply:
Good point thanks for noticing.